mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/blob/BucketNameGenerator.java

```diff
@@ -39,6 +39,6 @@ public BucketName currentBucket() {
         ZonedDateTime now = ZonedDateTime.now(clock);
         int month = now.getMonth().get(ChronoField.MONTH_OF_YEAR);
         int year = now.getYear();
-        return BucketName.of(String.format("deletedMessages-%d-%d-01", year, month));
+        return BucketName.of(String.format("deleted-messages-%d-%d-01", year, month));
```

Comment:
That doesn't match with the description in the commit... don't you mean instead S3 doesn't accept upper case characters?